engine/runtime/src/main/scala/org/enso/interpreter/instrument/job/ProgramExecutionSupport.scala

```diff
@@ -155,7 +155,11 @@ trait ProgramExecutionSupport {
     updatedVisualisations: Seq[Api.ExpressionId],
     sendMethodCallUpdates: Boolean
   )(implicit ctx: RuntimeContext): Option[Api.ExecutionResult] = {
-    ctx.executionService.getLogger.log(Level.FINEST, s"Run program $stack")
+    val logger = ctx.executionService.getLogger
+    logger.log(
+      Level.FINEST,
+      s"Run program $updatedVisualisations $sendMethodCallUpdates"
```

Comment:
What about something like this? So that the log messages are easier to understand
```suggestion
      s"Run program $updatedVisualisations sendMethodCallUpdates=$sendMethodCallUpdates"
```

Reply:
Updated :+1: